spec/01-unit/04-prefix_handler_spec.lua

```diff
@@ -19,6 +19,11 @@ ffi.cdef([[
 ]])
 
 
+local escape_pattern = '(['..("%^$().[]*+-?"):gsub("(.)", "%%%1")..'])'
```

Comment:
Could we add some comments about this pattern? 
It is hard to know what it did.

Reply:
Sure